zproject/computed_settings.py

```diff
@@ -233,7 +234,8 @@ def get_dirs(self) -> List[str]:
 ZILENCER_ENABLED = 'zilencer' in INSTALLED_APPS
 CORPORATE_ENABLED = 'corporate' in INSTALLED_APPS
 
-TORNADO_PORTS = get_tornado_ports(config_file)
+if not TORNADO_PORTS:
+    TORNADO_PORTS = get_tornado_ports(config_file)
```

Comment:
Would it be simpler to have `TORNADO_PORTS` to be defined using `get_tornado_ports` in `default_settings.py` , and then not have a line here at all?

Reply:
I was tempted to do that, but nothing in `default_settings.py` currently reads from `zulip.conf` -- it only has access to `PRODUCTION`, `DEVELOPMENT`, and `get_secret`.  Clearly, the former two are based on `zulip.conf` contents, so this isn't a hard line.  But it did seem like a departure from essentially just hard-coded values in `default_settings.py`